contracts/CO30_Rob_TuonelaTwoStep.hellanmaa/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,10 @@
+<font color='#E3D310'>PRIMARY</font>
+Destruction of the enemies ground radar installation
+
+Destruction of atleast half of the enemies artillery pieces
+
+<font color='#595305'>SECONDARY</font>
+Destruction of all enemy artillery assets
+
+Destruction of the enemies artillery ammunition depot
```

Comment:
We keep these as short and to the point as possible, in the form of actions and with `-`

```suggestion
- Destroy the ground radar installation
- Destroy atleast half of the artillery pieces

<font color='#595305'>SECONDARY</font>
- Destry all artillery assets
- Destroy the artillery ammunition depot
```